### PR TITLE
error on missing dependencies & use docker ZK-shell

### PR DESCRIPTION
**Pull request by brianbushree**
### what
1. check dependencies before starting `openssl docker-compose docker confluent jq`
2. use docker zk-shell command

### why
1. fail fast 🙂 
2. don't require users to modify their path variable to include `zookeeper-shell`

### CODE REVIEW

**Review by ybyzek — APPROVED** [review]
LGTM, pending approval from @stan-confluent .  I haven't run the Docker/RBAC demo yet

**Review by stan-is-hate — APPROVED** [review]
Thanks Brian! Looks good!

**Pull request merged**
